test/test_t1.py

```diff
@@ -56,6 +59,51 @@ def test_t1_end2end(self):
         for exp_res, load_res in zip(exp_res, load_res):
             self.analysis_result_equiv(exp_res, load_res)
 
+    def test_t1_measurement_level_1(self):
+        """
+        Test T1 experiment using a simulator.
+        """
+
+        ns = 1e-9
+        mu = 1e-6
+        t1 = [45 * mu, 45 * mu]
+
+        # delays
+        delays = np.logspace(1, 11, num=20, base=np.exp(1))
+        delays *= ns
+        delays = np.insert(delays, 0, 0)
+        delays = np.append(delays, [t1[0] * 2, t1[0] * 3])
```

Comment:
In the tutorial you've removed the last point, is there a reason not to do it also here?

Reply:
My mistake. It seems like it didn't converge so I changed (in the tests also) the code to have more data points and added only 1 extreme delay.
```python
...
# delays
delays = np.logspace(1, 11, num=23, base=np.exp(1))
delays *= ns
delays = np.insert(delays, 0, 0)
delays = np.append(delays, [t1[0] * 3])
...
```